web/src/views/Search/SearchSidebar.vue

```diff
@@ -207,7 +223,25 @@ export default defineComponent({
         <span v-if="isLoading">
           Loading results...
         </span>
-        <span v-else>Found {{ resultsCount }} results.</span>
+        <span v-else>Found {{ resultsCount }} samples.
+          <v-tooltip
+            bottom
+            open-delay="600"
+          >
+            <template #activator="{ on, attrs }">
+              <v-btn
+                class="mb-2"
+                icon
+                small
+                v-bind="attrs"
+                v-on="on"
+              >
```

Comment:
```suggestion
              <v-btn
                icon
                x-small
                v-bind="attrs"
                v-on="on"
              >
```
Feel free to ignore this suggestion if you disagree, but it looks like removing the class centers the icon with the text and it did seem to me like the `small` size was just a bit too big.

Reply:
I am fine with this change, I went back and forth on exactly this and just picked one. Really was a coin toss for me.